scripts/enable-resource-locking.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e
 
-JSONPATH="[?(contains(type, 'Storage') || contains(type, 'KeyVault') || contains(type, 'SQL') || contains(type, 'Insights') || contains(type, 'ManagedIdentity'))].[resourceGroup]"
+JSONPATH="[?(contains(type, 'Storage') && tags.\"databricks-environment\" != 'true' || contains(type, 'KeyVault') || contains(type, 'SQL') || contains(type, 'Insights') || contains(type, 'ManagedIdentity'))].[resourceGroup]"
```

Comment:
As discussed we might need a generic solution for all the managed resource groups which need to be skipped when applying locks.

Reply:
Currently cannot see a valid property I would consider for this, will see if it appears again on a different type of resource.